include/unk_0200B358.h

```diff
@@ -11,19 +11,19 @@
 
 #include "enums.h"
 
-StringFormatter * sub_0200B358(u32 param0);
+StringFormatter * StringFormatter_New(u32 param0);
 StringFormatter * sub_0200B368(u32 param0, u32 param1, u32 param2);
 void sub_0200B3F0(StringFormatter * param0);
 void sub_0200B48C(StringFormatter * param0, u32 param1, const Strbuf *param2, u32 param3, BOOL param4, u32 param5);
-void sub_0200B498(StringFormatter * param0, u32 param1, const TrainerInfo * param2);
+void StringFormatter_FormatPlayerName(StringFormatter * param0, u32 param1, const TrainerInfo * param2);
```

Comment:
Do these functions Format or Buffer?

We already have BufferNickname, so we should decide on a terminology.

Reply:
Does the message formatter buffer things too? Cause it might be buffer if that's the case

Reply:
tl;dr lhea is doing string formatting stuff next- we need to decide on Buffer vs Format terminology,  maybe make it more clear what the formatter does, etc